services/data/postgres_async_db.py

```diff
@@ -53,7 +52,7 @@ def __init__(self, name='global'):
         tables.append(self.metadata_table_postgres)
         self.tables = tables
 
-    async def _init(self, db_conf: DBConfiguration):
+    async def _init(self, db_conf: DBConfiguration, create_triggers=False):
```

Comment:
will having this default to false cause problems with some deployments? I remember talks about running the ui_backend on a read replica, which would mean that the metadata service would be responsible for creating the triggers on the write master?

Reply:
Excellent point, might be best to put this behind feature flag after all.